src/js/common/components/LoginModal.jsx

```diff
@@ -54,10 +72,10 @@ class LoginModal extends React.Component {
 
   handleLogin(e) {
     e.preventDefault(); // Don't try to submit the page
+    this.setState({ loginButtonClicked: true });
     const loginUrl = this.props.user.login.loginUrl;
```

Comment:
could be shortened to:
const { user.login.loginUrl: loginUrl, onUpdateLoginUrl } = this.props;